commit/solvers.py

```diff
@@ -121,6 +121,30 @@ def init_regularisation(commit_evaluation,
     regularisation['lambdaISO'] = float( lambdas[2] )
 
     # Solver-specific fields
+    # Check if idxs in groups need to be updated
+    if(0 in commit_evaluation.DICTIONARY['TRK']['kept']):
```

Comment:
Change to `if ( 0 in commit_evaluation.DICTIONARY['TRK']['kept'] ) :`

Reply:
I'll remove the "(  )" and the space before ":"